config/locales/pt.yml

```diff
@@ -1,23 +1,56 @@
 pt:
```

Comment:
@mllocs is this correct? Please talk to Sara about the conflict with `pt_BR`

Reply:
Hi! (ehehe, I'm here too :) 
The pt translation is 100% complete, but I've just noticed that there might be an overlap occurring with the Brazilian crew... there are signs of recent edits in pt instead of pt_BR that have overridden my previous translation... 
I wrote to tneves (Brazilian translator) and sigmund (PT transifex team coordinator) to sort this out.

Reply:
@mkllnk maybe we can check Transifex permissions for `pt` and `pt_BR`

Reply:
Thomas Neves here he is: https://community.openfoodnetwork.org/u/thomaz and Sigmund I think it @sigmundpetersen

Reply:
@saritamoreira and @mllocs shall I add one of you as coordinator of `pt`?

Reply:
Hi @sigmundpetersen, yes, you can add me!